ui/app/components/clients/dashboard.js

```diff
@@ -24,8 +24,14 @@ export default class Dashboard extends Component {
     { key: 'entity_clients', label: 'unique entities' },
     { key: 'non_entity_clients', label: 'non-entity tokens' },
   ];
-  // TODO remove this adapter variable? or set to /clients/activity ?
-  adapter = this.store.adapterFor('clients/new-init-activity');
+  adapter = this.store.adapterFor('clients/activity');
+  // needed for startTime modal picker
```

Comment:
Claire, I missed that you removed these in an earlier PR. I need them for the startTime modal.

Reply:
Oh woops! That was totally an accident. Sorry about that